src/guides/v2.3/release-notes/release-notes-2-3-4-commerce.md

```diff
@@ -518,7 +514,7 @@ We have fixed hundreds of issues in the Magento 2.3.4 core code.
 
 <!--- ENGCOM-5632-->
 
-*  Links to a sitemap generated at  **Marketing** > **SEO & Search** > **sitemap** is are now  valid. Previously, Magento returned a 404 error when you clicked on the sitemap link. *Fix submitted by Sunil in pull request [23716](https://github.com/magento/magento2/pull/23716)*. [GitHub-23706](https://github.com/magento/magento2/issues/23706)
+*  An incorrect xml namespace url was removed from the generated sitemaps displayed at **Marketing** > **SEO & Search** > **sitemap**. *Fix submitted by Sunil in pull request [23716](https://github.com/magento/magento2/pull/23716)*. [GitHub-23706](https://github.com/magento/magento2/issues/23706)
```

Comment:
This was an incorrect description, previously it seemed to suggest that the links to the generated sitemap files listed in the backend were incorrect and now fixed.
But that wasn't the fix in this PR here, it was removing incorrect url's being used as xml namespaces in the generated sitemap xml files.